Updated Documentation and included Germany DE Version (#6)
opencultureagency/Repair-Kit: README.md

* **Personal Use**: A comprehensive and well-organized toolkit for anyone who wants a solid repair foundation at home, DIY and adaptation possible due to its **open source** collaborative nature
* `you name it` - add your application scenario via [ticket](https://github.com/opencultureagency/Repair-Kit/issues/new)

## Modules included
- Module 1 ([repair-m01](docs/repair-m01.md)): Essentials
- Module 2 ([repair-m02](docs/repair-m02.md)): Soldering
- Module 3 ([repair-m03](docs/repair-m03.md)): Mobile Devices
- Module 4 ([repair-m04](docs/repair-m04.md)): Basic Tools
- Extras ([repair-extras](docs/repair-extras.md)): Extras - packing & additional material 

![](assets/Repair-Kit-modules-extra-ready-demo.JPG)

In the future there are other modules planned as this modular approach is inspired by the [#ASKotec-Modules](https://github.com/opencultureagency/ASKotec-Modules) and repair specific examples are available in the repository:

|Module 05   `draft`   | Module 06   `draft`  | Module 07  `draft`   | Module XX   |
|     ---       |     ---       |     ---       |     ---       |

> | repair-m05    | repair-m06    | repair-m07    | `you name it` |

Reviewer: ```suggestion
| repair-m05 Mobile Soldering    | repair-m06 Bike Basics   | repair-m07 Wrenches   | `you name it` |
```

The initial design for this new Repair Kit approach was supported by [repami.de](https://repami.de), a network for quality repairs in Berlin.

> For all documentation we apply to the [CC-BY-SA 4.0](/LICENSE_CC_BY-SA.md) apart from the technical files via [/src/](/src/) and [/src/CAD](/src/CAD/) in specific we refer to [CERN-OHL-W-2.0-or-later](/LICENSE_cern_ohl_w_v2.txt) for better integration (this also includes all production related exports to the [/prod/](/prod/) directory.). 

Reviewer: ```suggestion
On all documentation we apply the [CC-BY-SA 4.0](/LICENSE_CC_BY-SA.md). Apart from that, on the technical files (in [/src/](/src/) and [/src/CAD](/src/CAD/) in specific) we apply the [CERN-OHL-W-2.0-or-later](/LICENSE_cern_ohl_w_v2.txt) for better integration (this also includes all production related exports to the [/prod/](/prod/) directory.). 
```
